### PR TITLE
Migrations, Windows setup instructions, README updates

### PR DESCRIPTION
**Pull request by Ollie1700**
Hi there, I found your project on "Up for grabs" and thought it was a cool concept.

I have worked with Laravel a lot but never a version as old as 4.2, so I thought it'd be a cool challenge whilst ticking off some of your issues along the way.

To start, I did a lot of experimentation to get this working on Windows and I have now added appropriate instructions on the README to get this up and running on a Windows system. Long story short, Laravel 4.2 is not compatible with PHP v7+ due to an extension called `mcrypt` (that Laravel 4.2 requires) being totally removed from PHP 7. I did manage to find an easy enough workaround which is outlined in the README.

Secondly, I used your SQL dump file as a reference and converted all of the raw SQL into Laravel artisan migrations. This just makes sense, as since this is a Laravel project there is no reason for there to be a manual SQL dump to import - we can simply take advantage of the Eloquent system and set up all the tables that way.

Whilst converting the dump to migrations, I also noticed that there were several redundant timestamp fields - specifically ones that referenced things like when a resource was created. I have removed these redundant fields and updated all of their instances in blade templates and controllers with Laravel's default `updated_at` and/or `created_at` timestamp fields. It makes sense to use these over our own timestamp fields as Laravel will automatically update the `updated_at` fields whenever a change is made.

There were also some issues with running artisan commands when `app/storage` folders were not created. So I've removed `app/storage` from the `.gitignore`, created all necessary folders within `app/storage` and individually added `.gitignore`s to those folders so that the folders are part of the initial clone, but everything within them that is generated during development is ignored still. Hopefully that makes sense.

Anyway, I know this is a long message for a PR, but I just thought I'd outline everything in as much detail as possible. A very interesting project, was fun to work on and experiment with 😄 

Let me know if you had any questions at all, or if you wanted me to fix anything.

**EDIT:** Forgot to add that this PR also fixes Issue #31

### CODE REVIEW

File: app/controllers/BooksController.php [hunk 1]
```diff
@@ -19,13 +19,13 @@ public function index()
 
 		$book_list = Books::select('book_id','title','author','description','category_id')
 			->orderBy('book_id');
-		
+
```
Comment (prabhakar267) [inline app/controllers/BooksController.php]: Please remove all the files which have only spacing issues

**Comment by prabhakar267** [comment]
@Ollie1700 Thanks a lot for the PR. A lot of people had been asking me to add instructions for Windows setup and I didn't have enough resources to understand the Windows setup. I have had an email asking for the same almost once every 2 weeks. 

Secondly migrations was something I had been wishing to do for a long long time myself but never got time to add them myself. Again thanks for taking care of that.
Just one minor comment on the PR itself, you added some files which had only spacing changes, if you could remove them from your commit, I could merge your code.

Surely adding the Windows instructions is going to help a lot of people out there. 😄

**Comment by Ollie1700** [comment]
Thanks! Glad you liked the changes.

The `BooksController.php` actually has one line of code change at line 151. I removed the manual adding of the timestamp for the book (since Laravel does this automatically). As for the whitespace changes - this is because my Atom removes all unused whitespace to save file space. If this is an issue let me know as I'll have to go back through and re-add the whitespace.

Thanks 😄

**Comment by Ollie1700** [comment]
Thanks for accepting! Let me know if you have any questions at all.

**Pull request merged**